connectors/kafka-connector/src/test/scala/zio/connect/kafka/LiveKafkaConnectorSpec.scala

```diff
@@ -0,0 +1,82 @@
+package zio.connect.kafka
+
+import org.apache.kafka.clients.producer.ProducerRecord
+import zio._
+import zio.connect.kafka._
+import zio.connect.kafka.KafkaConnector.NewTopic
+import zio.kafka.KafkaTestUtils
+import zio.kafka.KafkaTestUtils._
+import zio.kafka.embedded.Kafka
+import zio.kafka.admin.AdminClient
+import zio.stream._
+import zio.test._
+import zio.test.Assertion._
+import zio.test.TestAspect._
+
+object LiveKafkaConnectorSpec extends ZIOSpecDefault {

```

Comment:
We should have a KafkaConnectorSpec that only depends on a KafkaConnector layer. And this LiveKafkaConnectorSpec can then only extend it and provide a LiveKafkaConnector. (smth like here - https://github.com/zio/zio-connect/tree/master/connectors/s3-connector/src/test/scala/zio/connect/s3)

Reply:
Yes, I tried that but couldn't make it work because I need different consumers for every test. That's how zio-kafka does it in its tests. I'll try it again but first I wanted to make some progress to make sure I'm on the right track :)